pytest_tests/testsuites/object/test_object_lock.py

```diff
@@ -18,17 +18,17 @@
     OBJECT_NOT_FOUND,
 )
 from neofs_testlib.shell import Shell
+from node_management import drop_object
 from pytest import FixtureRequest
 from python_keywords.container import create_container
 from python_keywords.neofs_verbs import delete_object, head_object, lock_object
+from storage_policy import get_nodes_with_object
 from test_control import expect_not_raises, wait_for_success
 from utility import parse_time, wait_for_gc_pass_on_storage_nodes
 
-import steps
 from helpers.container import StorageContainer, StorageContainerInfo
 from helpers.storage_object_info import LockObjectInfo, StorageObjectInfo
 from helpers.wallet import WalletFactory, WalletFile
-from steps.cluster_test_base import ClusterTestBase
```

Comment:
Why? it's used in 122 line

Reply:
it's duplicated from line 7:
`from cluster_test_base import ClusterTestBase`